### ISSUE TITLE
Pipes in href attributes are encoded by DOMDocument

### ISSUE THREAD

**Issue by schlessera** [ampproject/amp-toolbox-php#73]
When pipes are being used in `href` attributes, like it is the case for Google Fonts links, then the DOMDocument seems to encode these pipes, making them ugly to read.

<img width="916" alt="_ 2021-02-19 at 9 26 46 AM p-toolbox-php" src="https://user-images.githubusercontent.com/83631/108485401-c6256d80-7294-11eb-8c3d-393eaf2a1ce3.png">

**Comment by westonruter** [comment]
Just a cosmetic issue though, right? Functionally equivalent.

**Comment by schlessera** [comment]
Yes, purely cosmetic. Again something that makes testing more difficult across platforms, though. The above is the output from a PHPUnit test failing against the NodeJS spec test suite.
